blog/2023-01-10-pulsar-2022-year-in-review.md

```diff
@@ -0,0 +1,106 @@
+
+Before:
+
+![](/img/pulsar-website-before.png)
+
+After:
+
+![](/img/pulsar-new-website-after-1.png)
+
+![](/img/pulsar-new-website-after-2.png)
+
+Many community members [participated in the discussion and provided valuable feedback](https://github.com/apache/pulsar/issues/15538#issuecomment-1125602899). Currently, the Pulsar community is still working to improve the reader experience by making key information more visible and accessible.
+
+## Key events recap
+
+In 2022, the Apache Pulsar community put together a number of meetups and events across the globe to share the latest messaging and streaming technologies. Among others, two Pulsar Summits received the most attention from community members.
+
+- [Pulsar Summit San Francisco 2022](https://streamnative.io/blog/community/2022-08-25-pulsar-summit-sf-2022-community-event-recap/): This event marked the first-ever, in-person Pulsar Summit in North America with 200+ attendees. It featured 20 speakers from companies like Google, AWS, and Yahoo!, 5 keynotes on Apache Pulsar and other streaming technologies, and 12 breakout sessions on tech deep dives, use cases, and ecosystem talks. [Watch the sessions](https://streamnative.io/pulsar-summit-on-demand/).
+- [Pulsar Summit Asia 2022](pathname:///blog/2022/12/01/pulsar-summit-asia-2022-recap/): With 1,500+ registrations and 40,000+ views globally, this two-day virtual event gathered 41 speakers giving 3 keynote addresses and 36 sessions. They came from companies like ByteDance, Huawei, Tencent, Nippon Telegraph and Telephone Corporation (NTT), Yum China, Netease, vivo, Nutanix, and StreamNative. [Watch the sessions](https://www.youtube.com/playlist?list=PLqRma1oIkcWgeNb3kgzqFyg5sywH8S4yy).
+
+To stay up to date with the latest events, you can join the following meetups or groups:
+
+- [New York City Apache Pulsar Meetup](https://www.meetup.com/new-york-city-apache-pulsar-meetup/)
+- [SF Bay Area Apache Pulsar Meetup](https://www.meetup.com/sf-bay-area-apache-pulsar-meetup/)
+- [Netherlands Apache Pulsar Meetup](https://www.meetup.com/netherlands-apache-pulsar-meetup/)
+- [Japan Pulsar User Group](https://japan-pulsar-user-group.connpass.com/)
+
+## Community growth
+
+The development of the Pulsar community would not be possible without our contributors. Among them, Pulsar Committers and PMC members have taken the lead in contributing to and promoting the project. In 2022, many new faces joined the community while we also welcomed old friends taking more responsibility. Let’s take a look at the Pulsar community by numbers.
+
+- **598** global contributors for the Pulsar main GitHub repository
+- **12.1K+** GitHub stars
+- **3.2K+** forks
+- **16** new Committers
+- **5** new PMC members
```

Comment:
1. The NO. here is a bit confusing. Some are `total NO.` while others are `NO. of last year`. Does it make sense to add the `total NO. of committers and PMC members`? 

In this way, we can reduce users' confusion (data is consistent) and show the whole picture to them.

2. Does it make sense to add the `NO. of Pulsar releases` (show activeness and long history) and the `NO. of ecosystem plugins` (show diversity and compatibility)? 

These data can give users more trust and confidence in Pulsar.

Reply:
1. This is a summary blog for 2022. The only reason I added a total number is that I am not sure the exact number of growth from last year so I added the total number. @tuhaihe If you have the number please let me know. Thanks.
2. For releases, people can find the info in the sections below. For ecosystem tools, you can have a look at the discussions in this pr. It is difficult to include all tools as we may have different vendors developing them.